public/index.html

```diff
@@ -12,7 +12,7 @@
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <!-- <link rel="manifest" href="%PUBLIC_URL%/manifest.json" /> -->
```

Comment:
Can this be removed instead?

Reply:
Go with what Lewis has commented.

https://developer.mozilla.org/en-US/docs/Web/Manifest